test/dialect/mysql/test_match_expression.py

```diff
@@ -0,0 +1,103 @@
+from sqlalchemy import exc
```

Comment:
these are still compiler tests and can be in dialects/mysql/test_compiler.py

Reply:
done